bcftools-1.3.1/Makefile

```diff
@@ -0,0 +1,145 @@
+.PHONY: clean debian debclean
+
+# external commands used
+CURL  := /usr/bin/curl
+TAR   := /bin/tar
+CP    := /bin/cp
+RM    := /bin/rm
+MV    := /bin/mv
+MKDIR := /bin/mkdir
+ECHO  := /bin/echo
+AR    := /usr/bin/ar
+TEST  := /usr/bin/test
+
+# basic workspace directories
+WORK_DIR         := /build
+BASE_INSTALL_DIR := $(WORK_DIR)
+RESOURCES_DIR    := $(WORK_DIR)/resources
+RELEASE_DIR      := /release
+
+# source code information
+BCFTOOLS_VERSION    := 1.3.1
+BCFTOOLS_URL        := https://github.com/samtools/bcftools/releases/download/$(BCFTOOLS_VERSION)/bcftools-$(BCFTOOLS_VERSION).tar.bz2
+BCFTOOLS_ZIP        := $(RESOURCES_DIR)/bcftools-$(BCFTOOLS_VERSION).tar.bz2
+BCFTOOLS_OUTPUT_DIR := $(WORK_DIR)/bcftools-1.3.1
+BCFTOOLS            := $(WORK_DIR)/bin/bcftools
+
+# source code information
+HTSLIB_VERSION    := 1.3.2
+HTSLIB_URL        := https://github.com/samtools/htslib/releases/download/$(HTSLIB_VERSION)/htslib-$(HTSLIB_VERSION).tar.bz2
+HTSLIB_ZIP        := $(RESOURCES_DIR)/htslib-$(HTSLIB_VERSION).tar.bz2
+HTSLIB_OUTPUT_DIR := $(WORK_DIR)/htslib-1.3.2
+TABIX             := $(WORK_DIR)/bin/tabix
+BGZIP             := $(WORK_DIR)/bin/bgzip
+
+# debian packaging related variables
+DEB_BUILD_DIR       := $(WORK_DIR)/deb-build
+UBUNTU_EDITION      := $(shell . /etc/lsb-release; $(ECHO) $$DISTRIB_RELEASE)
+DEB_RELEASE_VERSION := 1ubuntu$(UBUNTU_EDITION)
+DEB_PKG             := ccdg-bcftools-$(BCFTOOLS_VERSION)_$(BCFTOOLS_VERSION)-$(DEB_RELEASE_VERSION).deb
+DEB_PKG_PATH        := $(RELEASE_DIR)/$(DEB_PKG)
+DEB_BASE_INSTALL    := /opt/ccdg/bcftools-$(BCFTOOLS_VERSION)
+
+# DEBIAN CONTROL FILE ##########################################################
+define debian_control
+Package: ccdg-bcftools-$(BCFTOOLS_VERSION)
+Architecture: amd64
+Section: science
+Maintainer: Indraniel Das <idas@wustl.edu>
+Priority: optional
+Depends: libc6, libcurl3, zlib1g, ca-certificates
+Description: bcftools-$(BCFTOOLS_VERSION) plus htslib-$(HTSLIB_VERSION) for the CCDG pipeline
+Version: $(BCFTOOLS_VERSION)-$(DEB_RELEASE_VERSION)
+endef
+export debian_control
+
+# DEBIAN POSTRM FILE ###########################################################
+define debian_postrm
+#!/bin/bash
+
+BASE=$(DEB_BASE_INSTALL)
+
+if [ -e $${BASE} ]; then
+    $(RM) -rfv $${BASE}
+fi
+
+endef
+export debian_postrm
+
```

Comment:
Minor nitpick detail:  could a header like:
```
# BCFTOOLS EXECUTABLE DEBIAN WRAPPER ##########################################
```
be added right before the `define bcftools_wrapper` statement.

I find it helps when quicking scanning the code again later in the future.